x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -171,39 +163,54 @@ export const getNetworkTopNFlowColumns = (
               expression={`network.direction: ${escapeQueryValue(direction)}`}
               type={type}
             >
-              {defaultToEmptyTag(direction)}
+              <>
+                {defaultToEmptyTag(direction)}
+                {index < directions.length - 1 ? '\u00A0' : null}
+              </>
             </AddToKql>
           )),
   },
   {
+    field: 'node.network.bytes',
     name: i18n.BYTES,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }) => {
-      if (node.network && node.network.bytes) {
-        return numeral(node.network.bytes).format('0.000b');
+    sortable: true,
+    render: (bytes: number) => {
+      if (!isNil(bytes)) {
+        return numeral(bytes).format('0.000b');
       } else {
         return getEmptyTagValue();
       }
     },
   },
   {
+    field: 'node.network.packets',
     name: i18n.PACKETS,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }) => {
-      if (node.network && node.network.packets) {
-        return numeral(node.network.packets).format('0,000');
+    sortable: true,
+    render: (packets: number) => {
+      if (!isNil(packets)) {
+        return numeral(packets).format('0,000');
       } else {
         return getEmptyTagValue();
       }
     },
   },
   {
+    field: `node.${topNFlowType}.count`,
     name: getUniqueTitle(topNFlowType),
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }) => getOrEmptyTag(`${topNFlowType}.count`, node),
+    sortable: true,
+    render: (ipCount: number) => {
+      if (!isNil(ipCount)) {
```

Comment:
The type above this says it cannot be a null or undefined but this `isNil` check implies it can be.

Suggestion:

We should use the typing that says `render` can contain a `null` or `undefined` as in:

```ts
ipCount: number | null | undefined
```

Same question for the other renders and those types within this file that use null/undefined checks.